src/latch_cli/main.py

```diff
@@ -553,6 +553,14 @@ def execute(
     type=bool,
     help="Open the registered workflow in the browser.",
 )
+@click.option(
+    "--mark-as-release",
+    "-r",
```

Comment:
```suggestion
    "-m",
```
people might get confused bc `-r` used to mean remote